src/hadoop-ai/build/docker-executor.patch

```diff
@@ -26,8 +26,8 @@ index a044cb6..819c496 100644
    //containername
    public static final String DOCKER_IMAGE_PATTERN =
 -    "^(([\\w\\.-]+)(:\\d+)*\\/)?[\\w\\.:-]+$";
-+          "^(([\\w\\.-]+)(:\\d+)*\\/)?([\\w\\.-]+\\/)?[\\w\\.:-]+$";
- 
++          "^(([a-zA-Z0-9.-]+)(:\\d+)?/)?([a-z0-9_./-]+)(:[\\w.-]+)?$";
```

Comment:
why not use `\w`?

Reply:
It's used by officials, we don't intend to change it.